docs/concepts/pdp/overview.mdx

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 1
+title: Overview
+---
+
+# Permit's PDP (Policy Decision Point)
+
+A PDP is a network node responsible for answering authorization queries using policies and contextual data.
+Hence, PDPs should be highly available, performant and physically close to the querying services to minimize latency and ensure high availability.
+
+Permit's PDP essentially becomes your microservice for authorization, bundling together OPA, OPAL and an API Server.
+
+The PDP is easy to install via Docker image, and will ensure **zero-latency, great performance, high availability, and improved security**.
+Like all of Permit's customer deployed components, the [Permit PDP is open-source](https://github.com/permitio/PDP) and is available publicly from [Docker hub](https://hub.docker.com/r/permitio/pdp-v2).
```

Comment:
isn't this duplicated ?

Reply:
90% of the docs here are duplicate🙈 I think it fits here